internal/controller/postgrescluster/controller_test.go

```diff
@@ -342,59 +341,32 @@ spec:
 			//
 			// The "metadata.finalizers" field is also okay.
 			// - https://book.kubebuilder.io/reference/using-finalizers.html
-			//
```

Comment:
Highly recommend turning off whitespace differences to see what's actually going on here for anyone reviewing this